src/bindings/python/flux/hostlist.py

```diff
@@ -167,6 +201,6 @@ def copy(self):
         return Hostlist(handle=self.pimpl.copy())
 
 
-def decode(string):
+def decode(arg):
     """Decode a string in RFC 29 hostlist format to a Hostlist object"""
```

Comment:
should this comment be updated to string or iterable?

Reply:
Yeah, thanks. I'm actually not sure of Python idioms, if `hostlist.decode()` just directly calls `hostlist.Hostlist()` is it even useful to separately provide that function?

Reply:
good point.  can't speak for whats common, but since there is an `encode`, i think it's nice to have a `decode`, minimally so people can search for it.

Reply:
Ah good point! It doesn't do any harm at least...